src/main/java/medicaldoctor/ParentInterface.java

```diff
@@ -0,0 +1,14 @@
+/*
+ * To change this license header, choose License Headers in Project Properties.
+ * To change this template file, choose Tools | Templates
+ * and open the template in the editor.
+ */
+package medicaldoctor;
+
+/**
+ *
+ * @author Salmanhussain
+ */
+public interface ParentInterface {
```

Comment:
"Interface" doesn't seem to be needed in the name, try a more descriptive name? this is a parent of what kind of object?